contracts/Contribution.sol

```diff
@@ -18,23 +18,23 @@ contract Contribution is Controlled, TokenController {
 
   uint256 public minimum_investment;
 
-  uint256 public startBlock;
-  uint256 public endBlock;
+  uint256 public startTime;
+  uint256 public endTime;
 
-  uint256 public initializedBlock;
-  uint256 public finalizedBlock;
+  uint256 public initializedTime;
```

Comment:
For comunication with other contracts I also need `initializedBlock` and maybe `finalizedBlock`

Reply:
but with which ones?

Reply:
I already changed all variables in Exchange contract